Android/app/src/androidTest/java/tie/hackathon/travelguide/Tests/DemoTest.java

```diff
@@ -13,7 +15,18 @@
 public class DemoTest extends EspressoTestBase {
 
     @Test
-    public void testDemo() throws Exception {
-        onView(withContentDescription("SIGNUP")).perform(click());
+    public void testIfSignUpIsSuccessful() throws Exception {
+        LoginPage.signUp("user", "0754211009", "test1234");
+        LoginPage.tapSignUp();
+        MainPage.signOut();
+        Assert.assertTrue("The login page is not displayed", Helpers.isIdDisplayed(R.id.ok_signup));
     }
-}
+
+    @Test
+    public void testIfLogInIsSuccessful() throws Exception {
+        LoginPage.tapLogIn();
+        LoginPage.logIn("0754211009", "test1234");
+        LoginPage.tapLogInButton();
+        Assert.assertTrue("The main page is not displayed", Helpers.isStringDisplayed(Strings.TRAVEL_MATE));
```

Comment:
Cum am zis si mai sus, metoda "isStringDisplayed" o apelezi intr-o metoda "isTravelMateDisplayed();", in screen-ul specific. Cand o vei folosi aici, va fi mult mai clar ce verifici.